s3/libirods_s3.cpp

```diff
@@ -5,32 +5,32 @@
 
 // =-=-=-=-=-=-=-
 // irods includes
-#include <msParam.h>
-#include <rcConnect.h>
-#include <rodsLog.h>
-#include <rodsErrorTable.h>
-#include <objInfo.h>
-#include <rsRegReplica.hpp>
-#include <dataObjOpr.hpp>
-#include <irods_stacktrace.hpp>
+#include <irods/msParam.h>
+#include <irods/rcConnect.h>
+#include <irods/rodsLog.h>
+#include <irods/rodsErrorTable.h>
+#include <irods/objInfo.h>
+#include <irods/rsRegReplica.hpp>
+#include <irods/dataObjOpr.hpp>
+#include <irods/irods_stacktrace.hpp>
 
 #ifdef USING_JSON
 #include <json/json.h>
 #endif
```

Comment:
any idea what this is?

Reply:
No, but it's been here since the very first commit in this repo nine years ago.
If we want to remove it or change it, should be a separate thing.

Reply:
does this look familiar @JustinKyleJames ?

Reply:
Created #2037 for this.